gradle/impl-gradle-embedded-archive/pom.xml

```diff
@@ -27,7 +27,7 @@
         <dependency>
             <groupId>org.gradle</groupId>
             <artifactId>gradle-tooling-api</artifactId>
-            <version>${version.gradle-tooling-api}</version>
+            <version>${version.org.gradle.gradle-tooling-api}</version>
```

Comment:
I still think you should go with package, not package.artifact. If we will have some automation in the future we should align this with what you will have in main pom.xml

Reply:
Having problem understanding what is the desired outcome for this comment. Can you provide expected version property for - for example - `arquillian-spacelift-api`?